script.js

```diff
@@ -0,0 +1,54 @@
+var word;
```

Comment:
How could we refactor to use an object-oriented approach? These global variables could be properties on an object